ziti/options.go

```diff
@@ -19,9 +20,11 @@ type Options struct {
 	RefreshInterval time.Duration
 	OnContextReady  func(ctx Context)
 	OnServiceUpdate serviceCB
+	HttpApiRequest  func(r *http.Request)
```

Comment:
an Options struct takes a function that does http requests? That seems -- odd to me? particularly without any sort of doc indicating why. Is this intentional? I don't see `HttpApiRequest` used in this PR, did I miss it?

Reply:
Oh, meant to remove that. That was first pass stuff.